docs/src/examples/components/Input/Variations/InputExampleTargeting.shorthand.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Grid, Input, Text } from '@stardust-ui/react'
+
+const propsForInput = { placeholder: 'Search...', id: 'inputId', role: 'checkbox' }
```

Comment:
nit: just to make the idea more visible, I would suggest to introduce comment like `this object contains properties that will be applied to the input`. But another point that I would like to make, just to share my thoughts on this - it seems to be really confusing to the client in a sense which prop belongs to which slot, and **we would need to not forget to maintain this example** with each change to this decision being made, as well as with each new prop being introduced. Frankly, to me it seems like a serious problem we should further think on (another PR, of course)